source/manual/alerts/data-sync.html.md

```diff
@@ -7,6 +7,11 @@ parent: "/manual.html"
 last_reviewed_on: 2019-05-02
 review_in: 6 months
 ---
+# Data Sync
+
+> **Note on AWS**
+>
+> In AWS the env-sync-and-backup was replaced by the [govuk_env_sync](/manual/govuk-env-sync.html)
```

Comment:
It might be worth going in to a bit more detail here, as env-sync-and-backup does work with AWS, specifically the Integration environment.

Reply:
Added in an additional sentence to elaborate

Reply:
:+1: